t/20-function-climb2mlwh.t

```diff
@@ -0,0 +1,78 @@
+use strict;
+use warnings;
+use Test::More tests => 3;
+use Test::Exception;
+use File::Temp qw/tempdir/;
+use File::Path qw/make_path/;
+use File::Copy;
+use File::Slurp;
+use Log::Log4perl qw/:levels/;
+use Moose::Meta::Class;
+use DateTime;
+use t::dbic_util;
+
+my $dir = tempdir( CLEANUP => 0);
+
+use_ok('npg_pipeline::function::climb2mlwh');
+
+# Create dummy command file
+my $exec = join q[/], $dir, 'wsi-npg-ssh-restricted';
+open my $fh1, '>', $exec or die 'failed to open file for writing';
+print $fh1 qq[
+printf 'perl: warning: Setting locale failed.\\n'
```

Comment:
The warnings come back on STDERR so should not be mixed with the real data returned I think,